pkg/core/transaction/transaction.go

```diff
@@ -251,6 +257,15 @@ func (t *Transaction) GroupInputsByPrevHash() map[util.Uint256][]*Input {
 	return m
 }
 
+// GroupOutputByAssetID groups all TX outputs by their assetID.
+func (t *Transaction) GroupOutputByAssetID() map[util.Uint256][]*Output {
```

Comment:
getter not need pointer, replace with `func (t Transaction)`

Reply:
@im-kulikov , I don't get this point. Why it is better to use `func (t *Transaction)` instead of `func (t Transaction)`? We have many methods similar to `GroupOutputByAssetID` like:
`GroupInputsByPrevHash`, `Size`, `Hash`. Are these methods also `getter`?

Reply:
if you don't need to change anything in your method, you don't need pointer `*` on receiver

Reply:
:+1: